solution.txt

```diff
@@ -179,9 +176,15 @@ const numArrays = [
 
 // Complete Exercise 13 below...
 
-const lastVal = numArrays.length - 1;
+// var numList=//console.log(numArrays.indexOf)
+var numList = []
+ for (let i=0; i<numArrays.length ; i++)
+ {
+   if (numArrays.length-1 === numArrays.indexOf(numArrays[i])) {
+     numList.push(numArrays[i])
+   }
+ }
```

Comment:
this looks a little funky to me. the opening curly bracket after the `for ()` (in line 182) should  be at the end of line 181.